src/layers/CalloutCanvasLayer.ts

```diff
@@ -68,21 +67,21 @@ export class CalloutCanvasLayer extends CanvasLayer {
 
   onUpdate(event: OnUpdateEvent): void {
     super.onUpdate(event);
-    if (!this.data) {
-      this.callouts = null;
-    }
+
+    this.callouts = null;
```

Comment:
Assigning null to something isn't great, so we should look into alternatives as we move forward.

Reply:
I changed it to undefined so we at least have one consistent state for not having calculated it.